addons/udes_stock_picking_batch/models/stock_picking_batch.py

```diff
@@ -71,13 +71,13 @@ class StockPickingBatch(models.Model):
         required=False,
         help="Name of the batch from which this batch was derived",
     )
-    u_picking_zone_id = fields.Many2one(
```

Comment:
Q: Have we made sure we have changed all instances of `u_picking_zone_id` to the new field format? Looks like there are some still in `addons/udes_stock_refactoring/tests/test_refactoring.py`, may be others too. I might be wrong as i'm looking at my local code rather than SE-1770 though

Reply:
The field is in 2 models, in stock.picking.batch and stock.location. Renamed in stock.picking.batch model, i think i checked the remaining ones are in location. Hopefully havent forgot any

Reply:
Cool, just wanted to double check! Thanks